packages/block-editor/src/hooks/style.js

```diff
@@ -155,6 +168,11 @@ export const withBlockControls = createHigherOrderComponent(
 			),
 			<ColorEdit key="colors" { ...props } />,
 			<BlockEdit key="edit" { ...props } />,
+			hasPaddingSupport && (
+				<SpacingPanelControl key="spacing">
```

Comment:
Any reason for this component to be used in the "style" hook instead of directly in "PaddingEdit"? Do we expect any  other "spacing" hook to be added later?

Reply:
A potential spacing related hook may be `margin`.